extensions/amp-3q-player/0.1/test/test-amp-3q-player.js

```diff
@@ -42,13 +42,12 @@ describes.realWin('amp-3q-player', {
     }
     doc.body.appendChild(player);
     return player.build().then(() => {
-      const layoutPromise = player.layoutCallback();
+      player.layoutCallback();
```

Comment:
Curious: Would it be possible to return a promise here that resolves sooner than `player.layoutCallback()`? If so, the rest of the code needn't be changed.

Reply:
yeah that would work as well. nothing was using layoutPromise anyway.